extension/encoding/googlecloudlogentryencodingextension/extension_test.go

```diff
@@ -29,6 +30,8 @@ func newTestExtension(t *testing.T, cfg Config) *ext {
 }
 
 func TestHandleLogLine(t *testing.T) {
+	t.Parallel()
```

Comment:
I added it to the new test, felt like I may as well add it here for consistency.